exercises/grains/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef _GRAINS_H
+#define _GRAINS_H
+
+unsigned long square(int sq);
+unsigned long total(void);
```

Comment:
Personally I would prefer pulling in uint64_t for things that require fixed-width types. It might be worthwhile to make an issue to discuss.